modules/graph/loader/arrow_fragment_loader.h

```diff
@@ -189,10 +189,9 @@ GatherETables(Client& client,
     }
     std::unordered_map<std::string, std::string> meta_map;
     metadata->ToUnorderedMap(&meta_map);
-    grouped_batches[meta_map.at("label")]
-                   [std::make_pair(meta_map.at("src_label"),
-                                   meta_map.at("dst_label"))]
-                       .emplace_back(batch);
+    grouped_batches[meta_map["label"]][std::make_pair(meta_map.at("src_label"),
+                                                      meta_map.at("dst_label"))]
+        .emplace_back(batch);
```

Comment:
Are `"src_label"` and `"dst_label"` guaranteed to exist?

Reply:
These three key may not exists, however, a empty string will be created automatically and that will suffice.

Reply:
`at` will raise.